hibernate-core/src/test/java/org/hibernate/orm/test/jpa/metamodel/attributeInSuper/WorkOrder.java

```diff
@@ -8,15 +8,16 @@
 
 import java.io.Serializable;
 import java.util.Set;
+
 import jakarta.persistence.CascadeType;
-import jakarta.persistence.EmbeddedId;
+import jakarta.persistence.Id;
 import jakarta.persistence.OneToMany;
 import jakarta.persistence.OrderBy;
 
 @jakarta.persistence.Entity
 public class WorkOrder implements Serializable {
-	@EmbeddedId
-	private WorkOrderId id;
+	@Id
+	private Long id;
```

Comment:
Why this change?

Reply:
This test was weird, with a component annotated both `@Embeddable` and `@MappedSuperclass` that was extended by another `@Embeddable`. Before this was "ignored", though now that we have inheritance the semantics have changed.